docs/running-a-node/rocksdb-ledger-backend.md

```diff
@@ -0,0 +1,82 @@
+The node ledger currently uses LMDB (Lightning memory-mapped database) by default as the data store. As of _v20+_ the option to use RocksDB becomes available as an experimental option.
```

Comment:
Above the content add a callout box with a warning, something like this:

!!! warning "RocksDB is experimental, do not use in production"
    RocksDB is being included in V20.0 as experimental only and should be used in development environments only. Do not use in production environments. Future versions of the node may allow for production use of RocksDB.